content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -9,4 +9,339 @@ owner: istio/wg-networking-maintainers
+service/reviews labeled
+{{< /text >}}
+
+For any requests from any pods in ambient to the `reviews` service, the requests must go through the `reviews-svc-waypoint` for L7 processing and policy enforcement.
+
+### Configure a specific pod with its own waypoint
+
+Deploy a waypoint called `reviews-v2-pod-waypoint` for the `reviews-v2` pod:
+
+{{< text bash >}}
+$ istioctl experimental waypoint apply -n default --name reviews-v2-pod-waypoint --for workload
+waypoint default/reviews-v2-pod-waypoint applied
+{{< /text >}}
+
+Label the `reviews-v2` pod to use the `reviews-v2-pod-waypoint` waypoint:
+
+{{< text bash >}}
+$ kubectl label pod -l version=v2,app=reviews istio.io/use-waypoint=reviews-v2-pod-waypoint
+pod/reviews-v2-5b667bcbf8-spnnh labeled
+{{< /text >}}
+
+For any requests from any pods in ambient to the `reviews-v2` pod IP, the requests must go through the `reviews-v2-pod-waypoint`
+for L7 processing and policy enforcement.
+
+## Attach L7 policies to waypoint proxies {#attachl7policies}
+
+The following L7 policies are supported for waypoint proxy:
+
+|  Name  | Feature Status | Policy Attachment |
+| --- | --- | --- |
+| `HTTPRoute` | Beta | `parentRefs` |
+| `TCPRoute` | Beta | `parentRefs` |
+| `TLSRoute` | Beta | `parentRefs` |
+| `AuthorizationPolicy` | Beta | `targetRefs` |
+| `RequestAuthentication` | Beta | `targetRefs` |
+| `Telemetry` | Alpha | `targetRefs` |
+| `WasmPlugin` | Alpha | `targetRefs` |
+| `EnvoyFilter` | Alpha | `targetRefs` |
+
+### Attach a L7 policy to the entire waypoint proxy
+
+To attach a L7 policy to the entire waypoint, set `Gateway` as the `parentRefs` or `targetRefs` value, depending on your policy type.
+The example below shows how to apply an `AuthorizationPolicy` policy to the waypoint named `waypoint` for the `default` namespace:
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: viewer
+  namespace: default
+spec:
+  targetRefs:
+  - kind: Gateway
+    group: gateway.networking.k8s.io
+    name: waypoint
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        namespaces: ["default", "istio-system"]
+    to:
+    - operation:
+        methods: ["GET"]
+EOF
+{{< /text >}}
+
+### Attach a L7 policy to a specific service
+
+To attach a L7 policy to a specific service within the waypoint, set `Service` as the `parentRefs` or `targetRefs` value. The example below shows how to apply
+the `reviews` HTTPRoute to the `reviews` service in the `default` namespace:
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: reviews
+spec:
+  parentRefs:
+  - group: ""
+    kind: Service
+    name: reviews
+    port: 9080
+  rules:
+  - backendRefs:
+    - name: reviews-v1
+      port: 9080
+      weight: 90
+    - name: reviews-v2
+      port: 9080
+      weight: 10
+EOF
+{{< /text >}}
+
+## Debug your waypoint proxies
```

Comment:
(This will go to its own page too)

Reply:
Okay with that organization after the content merged